plenum/server/validator_info_tool.py

```diff
@@ -736,3 +744,10 @@ def _get_stop_stat(self):
                     "total_count": len(stops),
                 }
         return res
+
+    def _get_ledgers_updated_time(self)->dict:
+        return self._node.master_replica.get_ledgers_last_update_time()
+
+    def _is_updated_time_acceptable(self, updated_time):
+        current_time = self._node.utc_epoch()
+        return current_time - updated_time <= 2 * self._node.config.STATE_FRESHNESS_UPDATE_INTERVAL
```

Comment:
Please make `2` a parameter in config as well (something like ACCEPTABLE_FRESHNESS_INTERVALS_COUNT)